content/_guides/faq.md

```diff
@@ -59,6 +59,23 @@ to find when they're connected. There are also some robots, currently
 If you are unable to [connect to the network](/guides/connect) at all, you can
 contact staff by email: <support@libera.chat>.
 
+## What are Global Notices and Wallops, and can I opt out of them?
+
+`Global Notices` are network wide messages that staff can send to everyone on
+the network, and are reserved for important news, warnings, or explanations
+about network related things such as netsplits or downtime. You **cannot** opt
+out of `Global Notices`.
+
+`Wallops` are messages sent to everyone who has the +w [user mode](/guides/usermodes)
```

Comment:
```suggestion
**Wallops** are messages sent to everyone who has the `+w` [user mode](/guides/usermodes)
```

I suggest making this bold instead of code for the same reason as the initial definition of the term "global notice." Furthermore, the `+w` user mode *does* need to be marked up as code per our style guide on README.md.